app/docker/components/volumesNFSForm/volumesNFSFormController.js

```diff
@@ -0,0 +1,5 @@
+angular.module('portainer.docker')
+.controller('VolumesNFSFormController', [
+function () {
+  var ctrl = this;
```

Comment:
This controller looks to be pretty basic, actually as it's not doing anything, I think that you can remove it.